Source/WebKit/UIProcess/Cocoa/WKShareSheet.mm

```diff
@@ -333,7 +329,7 @@ - (void)presentWithShareDataArray:(NSArray *)sharingItems inRect:(std::optional<
     }];
 
 #if PLATFORM(VISION)
```

Comment:
We don't actually need this conditional right?

Reply:
Though it will probably cause issues with current EWS if we don't replace with an SDK version check... so never mind.

Reply:
Indeed :)